app.js

```diff
@@ -8,25 +8,24 @@ const router = require('./routes/basic');
 const bodyParser = require('body-parser');
 
 
-
+// For use the body parser
 app.use(express.json(), bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
+// Handle Routing
 app.use(router);
 
+// Database connnetion
 const startServer = async () => {
     mongoose.connect('mongodb://localhost:27017/test', { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
use all confidential url and values (like connection string, any password) from env variables